namui/src/namui/skia/web/canvas_kit/mod.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(unused_doc_comments)]
+#![allow(unused_doc_comments, dead_code)]
```

Comment:
It shows previous PR merged code.
I made ticket about this.
https://support.github.com/ticket/personal/0/1807703